### ISSUE TITLE
Lua error with fire mage in group (in LibClassicDurations)

### ISSUE THREAD

**Issue by treetip** [Tercioo/Plater-Nameplates-Classic#5]
```
Message: ...face\AddOns\Plater\libs\LibClassicDurations\core.lua:575: attempt to index local 'spellTable' (a nil value)
Time: Fri Aug  7 22:18:05 2020
Count: 2
Stack: ...face\AddOns\Plater\libs\LibClassicDurations\core.lua:575: attempt to index local 'spellTable' (a nil value)
[C]: ?
...face\AddOns\Plater\libs\LibClassicDurations\core.lua:575: in function `IngiteHandler'
...face\AddOns\Plater\libs\LibClassicDurations\core.lua:597: in function <...face\AddOns\Plater\libs\LibClassicDurations\core.lua:588>
(tail call): ?
(tail call): ?

Locals: 
```

Mentioned briefly in rgd87/LibClassicDurations#72 .

Is there a way to disable this via plater? or should i just edit the included LibClassicDurations function to a noop or something?

**Comment by cont1nuity** [comment]
I think it is best to open a ticket with LibClassicDurations.
I'll push an update to Plater as soon as I see a confirmation that this is fixed.

**Comment by treetip** [comment]
I'll mention it in the LibClassicDurations issue (changed the issue number, linked the wrong one initially)

**Comment by treetip** [comment]
Supposedly fixed in v64 of LibClassicDurations

**Comment by cont1nuity** [comment]
Fixed with 04ad2be
